core/test/audits/seo/is-crawlable-test.js

```diff
@@ -6,6 +6,8 @@
 
 import assert from 'assert/strict';
 
+import {expect} from 'expect';
```

Comment:
This is already a global right?

Reply:
yeah. i removed from other places too just now